runtime/queries/crystal/highlights.scm

```diff
@@ -1 +1,300 @@
-; inherits: ruby
+[
+  ","
+  ";"
+  "."
+  ":"
+  "*"
+  "**"
+] @punctuation.delimiter
+
+[
+  "alias"
+  ;; "alignof"
```

Comment:
These ones are commented out as they're supposed to work (and do in Zed) but didn't when I tested them

Reply:
(Same with a lot of other commented out things in this file)